TrainingHW1/TrainingHW1/Program.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+
+namespace TrainingHW1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+
+            byte MinByteValue = byte.MinValue;
```

Comment:
It's not necessary to create this variables. Values of types already stored in system libraries.